src/app/static/templates/data-exploration.ftl

```diff
@@ -6,6 +6,19 @@
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
     <!-- inject:css -->
     <!-- endinject -->
+    <script>
+    var host = window.location.hostname;
+    if(host != "localhost")
```

Comment:
Looks good to me, I guess we don't really want to show Ian the activities on preview, dev and etc. What is the implication of this tracking as regards GDPR ? Do we have to ask for users consent ?

Reply:
Could just have this if `host == naomi.unaids.org` but I wonder if we can get that out of config somewhere?

GDPR wise, looks not - and looks like even if we asked for consent it still wouldn't be. Basing this off https://piwik.pro/blog/is-google-analytics-gdpr-compliant/#does-collecting-visitors%E2%80%99-consent-solve-compliance-issues-with-google-analytics?

Reply:
Let's leave this PR open for now and discuss with UNAIDS & science